trino/trino.go

```diff
@@ -693,6 +712,19 @@ func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmt
 		hs = make(http.Header)
 		var ss []string
 		for _, arg := range args {
+			if strings.Contains(arg.Name, "Progress-Callback") {
```

Comment:
This condition is unnecessary, since you test for exact names in the following condition blocks.